README.md

```diff
@@ -3,8 +3,99 @@
 [![Build Status](https://img.shields.io/travis/18F/glossary/master.svg)](https://travis-ci.org/18F/glossary)
 [![Test Coverage](https://img.shields.io/codecov/c/github/18F/glossary/master.svg)](https://codecov.io/github/18F/glossary)
 
-A work in progress.
+# Getting started
+# About
+Add a simple glossary panel to your site to help your users understand jargon-y terms. As seen on [betaFEC](https://beta.fec.gov) and [EITI](https://useiti.doi.gov/).
 
+## Download
+### Via npm (coming soon)
+```
+npm install @18f/glossary-panel
```

Comment:
Oops, I misunderstood the guidance on this. Changed it to just `glossary-panel`.